### DISCUSSION TITLE
[ASA/ASE] Store death inventory in logs/refund package

### DISCUSSION THREAD

**Discussion by ScaredDonut** [gameserverapp/Platform#635] [Suggestions]
I am not sure if its possible without a mod but I've had some people die due to getting crashed into the world border by a dino without them being able to get their items back. Refunding all the items took forever so it'd be nice if there was a way to store what items they had into a log file or something like a shop package so it can be refunded with 1 click.

Thanks! :)

**Comment by mvaessen** [comment]
Hi, thanks for your suggestions. However, this is pretty out-of-scope for what GSA is set to do. I recommend posting this suggestion at  ARK's forum
